src/utils.js

```diff
@@ -202,6 +220,13 @@ go.utils = {
             .toUpperCase();         // capitalise
     },
 
+// ARRAY HELPERS
+
+    // creates a randomized array of shuffled values using Fisher-Yates shuffle
+    // ex. _.shuffle([1, 2, 3, 4]); → [4, 1, 3, 2]
+    randomize_array: function(array) {
```

Comment:
Given the simplicity of this function I'd rather its use was removed and just call `_.shuffle(array)` inline please